documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+= Debezium Connector for Db2 
+include::../_attributes.adoc[]
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+[NOTE]
+====
+Want to help us further hone and improve it? link:/docs/contribute/[Learn how].
+====
+
+toc::[]
+
+Debezium's Db2 Connector can monitor and record the row-level changes in the tables of a Db2 database.
+This connector was added in release Debezium 1.1.x.
+It is strongly inspired by the Debezium implementation of  SQL server, using a
+SQL based polling model of tables put into "capture mode". This uses the ASN libraries that come as a standard part of Db2 LUW and can be added (with an additional license) to Db2 zOS.
```

Comment:
Needs update as per the licensing discussion.

Reply:
Do you have a "boilerplate" text for this ?

Reply:
There's something similar for Oracle: https://github.com/debezium/debezium/blob/master/documentation/modules/ROOT/pages/connectors/oracle.adoc#deploying-a-connector

For DB2 this should not recommend to copy the driver to Connect lib dir, but rather the connector directory itself.

Reply:
Are you referring here specifically to the license for jdbc OR for the usage of the Db2 ASN services ?

Reply:
Here I'm referring to the IIDR license. I'm going to push a commit for this to this PR.